packages/artifact/__tests__/upload.test.ts

```diff
@@ -106,6 +107,17 @@ describe('Upload Tests', () => {
     )
   })
 
+  it('Create Artifact - Retention Less Than Min Value Error', async () => {
+    const artifactName = 'invalid-artifact-name'
```

Comment:
```suggestion
    const artifactName = 'valid-artifact-name'
```

The current name can cause another error to be thrown (there is another test for the artifact name), best to rule that out and use a valid name.